google-cloud-bigquery/lib/google/cloud/bigquery/convert.rb

```diff
@@ -60,6 +60,18 @@ def self.format_row row, fields
           end
         end
 
+        def self.is_int? value
```

Comment:
I would consider a different implementation with just a regex check, e.g. the following should be equivalent to your implementation, is more Ruby-like, and should also be a bit faster:

```ruby
def self.is_int? value
  /\A\s*[-+]?\d+\s*\z/.match? value.to_s
end
```